### ISSUE TITLE
Looking for a new maintainer

### ISSUE THREAD

**Issue by rcoedo** [prettier/prettier-emacs#49]
I'm not having the time to properly maintain this repository anymore. If anyone with enough experience wants to take over I would be happy to hand over the baton 😃 

@jscheid I've seen that you're doing a new prettier package for emacs, are you interested in becoming the new maintainer here?
 
@jlongster I'm mentioning you since this may interest you.

**Comment by jscheid** [comment]
Sorry but I'll pass. I'm planning to maintain the other package, but maintaining both seems a bit much (and somewhat redundant). Thanks for all your work on this one though.

**Comment by rcoedo** [comment]
Fair enough. Good look with your package!

**Comment by rcoedo** [comment]
@jscheid This is really not my package. I only published it to MELPA and maintained it with minor bug fixes and merging other's people patches. The package is the official one from the prettier organisation that once lived in the main prettier repository.

I think it would be nice for the emacs package ecosystem to try to avoid the confusing situation that we have with the javascript packages, where we have a `js-mode`, `js2-mode`, `js3-mode` and nobody knows what to use.

If I were you I think I'd try to contact somebody [from the prettier organisation](https://github.com/orgs/prettier/people). If this package is unmaintained but you commit to maintain yours it probably makes sense to officially point people out to your package instead. In this case you could even take the current `prettier-js` package name (we could write a migration guide).

If I find the time I'd be glad to help you with bug fixes or minor stuff 😄

**Comment by rcoedo** [comment]
Yes, you are right about that, with multiple language support `prettier` is probably a better name. If you get a response, please, let me know how it goes.

Cheers!

**Comment by jscheid** [comment]
@rcoedo it now got accepted on MELPA with package name `prettier`, which @jlongster thought was fine. Thanks again for all your work on this one.

**Comment by jcs090218** [comment]
@rcoedo Hi, does this repo still need a maintainer? I think I could afford some maintenance in my spare time. I wonder what would be the qualification to become a maintainer to this package? Thanks!

**Comment by david-alvarez-rosa** [comment]
Any updates on this?

**Comment by jacksonrayhamilton** [comment]
I too would be happy to help with maintaining this Emacs package.  I opened an issue in the main Prettier repo but haven't heard back yet: https://github.com/prettier/prettier/issues/17540

**Comment by fisker** [comment]
Hello, @jscheid are you interested to move https://github.com/jscheid/prettier.el to prettier org?
